app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/view/learning/LearningFragment.kt

```diff
@@ -222,29 +227,29 @@ class LearningFragment: Fragment() {
         return true
     }
 
+    private fun onImageToSortTouched(view: View, event: MotionEvent): Boolean {
+        val clipDataLabel = "My Clip Data"
```

Comment:
Might not be very useful, but for the sake of completness maybe make it related to the actual data?

Reply:
yep. this still dates from the prototyping phase of this fragment. Will fix